dev/build_release.py

```diff
@@ -654,6 +667,12 @@ def init_argument_parser(cls, parser):
                'Used to specify any custom behavior in the gradle builds.'
                'Argument is a file path relative to the directory this script is executed in.'
                'The default value assumes we run this script from the parent directory of spinnaker/spinnaker.')
+      parser.add_argument(
```

Comment:
ok, though ideally these should be named gcb_... to scope them to the gcb build module.

Reply:
Done